app/src/main/res/values-night/colors.xml

```diff
@@ -0,0 +1,178 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Everything above sounds good. The example makes lot of sense. Thanks.
Replying to last two questions:
1. themes should reference only `component_colors.xml` just to keep it consistent with `layout/*.xml` files as they will also be referencing only `component_colors.xml` file.
2. Yes, I agree there is inconsistency and started because of two things: (a.) by default strings creating by android like `app_name` used snake_case and (b.) the colors.xml file had three colors by default at start which are `colorPrimary`, `colorPrimaryDark`, `colorAccent` which followed lowerCamelCase format and because of this the confusion as well as inconsistency started. From now onwards lets make this final: **All files in values folder should follow snake_case naming convention**

Reply:
SGTM @rt4914. If you'd like, I think we can even make a regex check for this by prohibiting capitalization in certain contexts. I'm not sure if this'll have any false positives, but we can try it and see what existing cases are triggered:

```textproto
file_content_checks {
  file_path_regex: ".+?\\.xml"
  prohibited_content_regex: "name=\".+?[A-Z].+?\"
  failure_message: "All resource names must be written in snake_case."
}
file_content_checks {
  file_path_regex: ".+?\\.xml"
  prohibited_content_regex: "android:id=\".+?[A-Z].+?\"
  failure_message: "All layout IDs must be written in snake_case."
}
```

WDYT?